src/html-entrypoint.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { tokenize } = require('simple-html-tokenizer');
+const fs = require('fs');
+const path = require('path');
+const pattern = /<fastboot-script[^>]*><\/fastboot-script>\n*/g;
+
+function htmlEntrypoint(distPath, htmlPath) {
+  let html = fs.readFileSync(path.join(distPath, htmlPath), 'utf8');
+
+  // all the scripts we want to run go into appFiles. We don't use vendorFiles.
+  // The distinction doesn't matter here, as long as the scripts all stay in the
+  // right relative order.
+  let appFiles = [];
+
+  for (let token of tokenize(html)) {
+    if (token.type === 'StartTag' && ['script', 'fastboot-script'].includes(token.tagName)) {
+      for (let [name, value] of token.attributes) {
+        if (name === 'src') {
+          appFiles.push(path.join(distPath, value));
```

Comment:
Based on the testing within our application this should skip external scripts like

```
<script src="https://cdn.onesignal.com/sdks/OneSignalSDK.js" defer=""></script>
```

which results in FastBoot error

```
Error: ENOENT: no such file or directory, open '/var/folders/4z/h7wb5mp50sxbqpt8bdt_nlg40000gn/T/broccoli-60630hW7MihRllKHG/out-666-packager_runner_embroider_webpack/https:/cdn.onesignal.com/sdks/OneSignalSDK.js'
```

Reply:
That is a good point, we already have this policy in embroider itself when it consumes the same HTML. We should only extract relative URLs, and we should probably keep the earlier suggestion of a data-fastboot-ignore attribute.